flatliners/influxdbstorage.py

```diff
@@ -0,0 +1,20 @@
+from .baseflatliner import BaseFlatliner
+
+from influxdb import InfluxDBClient
+import time
+
+class InfluxdbStorage(BaseFlatliner):
+    def __init__(self):
+        super().__init__()
+
+
+    def on_next(self, x):
+        """ update l2 distance between cluster vector and baseline vector
+        """
+        client = InfluxDBClient(host='localhost', port=8086)
+        client.switch_database('flatliner')
+        client.write_points([{"measurement": "clusterdata",
+        "tags": {
+            "clusterID": x['cluster']
+        },
+        "fields": x}], time_precision='ms')
```

Comment:
does `x` contain a timestamp?

Reply:
x does not contain a timestamp. Looking at the output it seems like influxDB generates a timestamps for each record. 

are you suggesting that `time_precision='ms'` is not needed or that we need to add a timestamp to the record?

Reply:
we need a timestamp that aligns with the time that from the metrics used to generate this.
In the end we want to show a graph of the input data next to the weirdness score

but you can do this in a follow up PR